src/colorizer/Dataset.ts

```diff
@@ -449,16 +453,18 @@ export default class Dataset {
     }
 
     // Keep original sorting order of features by inserting in promise order.
-    featureResults.forEach((result) => {
-      const featureValue = this.getPromiseValue(result, "Failed to load feature: ");
+    featureResults.forEach((result, index) => {
+      const featureValue = this.getPromiseValue(result, `Failed to load feature ${index}`);
       if (featureValue) {
         const [key, data] = featureValue;
         this.features.set(key, data);
       }
     });
 
-    if (this.features.size === 0) {
-      throw new Error("No features found in dataset. Is the dataset manifest file valid?");
+    if (this.features.size !== manifest.features.length) {
+      console.warn(
+        "One or more features could not be loaded. This may be because of an unsupported format or a missing file. See console for details."
+      );
```

Comment:
"see console" when the user is already in the console? 🤔

Reply:
Ah, yeah. I think I was originally debating between showing this as a full error or just a console warning. Ideally, I think I need `Dataset.open` to report issues via warnings or banners on the viewer page. Ended up opening  https://github.com/allen-cell-animated/timelapse-colorizer/issues/411 to capture this.